integration/kubernetes/k8s-nested-configmap-secret.bats

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bats
+#
+# Copyright (c) 2021 IBM Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+load "${BATS_TEST_DIRNAME}/../../.ci/lib.sh"
+load "${BATS_TEST_DIRNAME}/tests_common.sh"
+
+setup() {
+	export KUBECONFIG="${KUBECONFIG:-$HOME/.kube/config}"
```

Comment:
This initialization can be improved on *all tests* as suggested in https://github.com/kata-containers/tests/issues/4242 (commenting here just to advertise the issue :)